src/server_manager/web_app/digitalocean_account.ts

```diff
@@ -60,7 +60,8 @@ export class DigitalOceanAccount implements digitalocean.Account {
     const regions = await this.digitalOcean.getRegionInfo();
     return regions.map(info => ({
       cloudLocation: new digitalocean.Region(info.slug),
-      available: info.available && info.sizes.indexOf(MACHINE_SIZE) !== -1
+      available: info.available && info.sizes.indexOf(MACHINE_SIZE) !== -1,
+      lowerCost: false,  // All DO regions have the same prices.
```

Comment:
This is the wrong semantics. They should all have this set to true. You can revert this file.

Reply:
Reverted